src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenCommonParameters.cs

```diff
@@ -25,7 +26,18 @@ internal class AcquireTokenCommonParameters
 
         public void AddApiTelemetryFeature(ApiTelemetryFeature feature)
         {
-            _apiTelemetry[MatsConverter.AsString(feature)] = "true";
+            _apiTelemetry[MatsConverter.AsString(feature)] = TelemetryConstants.True;
+        }
+
+        public void AddApiTelemetryFeature(ApiTelemetryFeature feature, bool isTrue)
```

Comment:
Is this feature (tracking the value of some flags) needed? It seems like a scope creep. Please remove if.